test/fail_compilation/fail117.d

```diff
@@ -1,8 +1,10 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail117.d(35): Error: expression has no value
-fail_compilation/fail117.d(36): Error: expression has no value
+fail_compilation/fail117.d(37): Error: expression `foo.mixin MGettor!(a) geta;
+` is `void` and has no value
+fail_compilation/fail117.d(38): Error: expression `foo.mixin MGettor!(b) getb;
+` is `void` and has no value
```

Comment:
Test 'fail_compilation\fail117.d' failed: 
expected:
----
fail_compilation\fail117.d(37): Error: expression `foo.mixin MGettor!(a) geta;
` is `void` and has no value
fail_compilation\fail117.d(38): Error: expression `foo.mixin MGettor!(b) getb;
` is `void` and has no value
----
actual:
----
fail_compilation\fail117.d(37): Error: cannot implicitly convert expression `foo.mixin MGettor!(a) geta;
` of type `fail117.Foo` to `int`
fail_compilation\fail117.d(38): Error: cannot implicitly convert expression `foo.mixin MGettor!(b) getb;
` of type `fail117.Foo` to `int`
----
diff:
----
-fail_compilation\fail117.d(37): Error: expression `foo.mixin MGettor!(a) geta;
-` is `void` and has no value
-fail_compilation\fail117.d(38): Error: expression `foo.mixin MGettor!(b) getb;
-` is `void` and has no value
+fail_compilation\fail117.d(37): Error: cannot implicitly convert expression `foo.mixin MGettor!(a) geta;
+` of type `fail117.Foo` to `int`
+fail_compilation\fail117.d(38): Error: cannot implicitly convert expression `foo.mixin MGettor!(b) getb;
+` of type `fail117.Foo` to `int`
----

Reply:
The output is fine but it needs #12542 first.